### PR TITLE
[OBSDEF - 10007] Changed placeholder text and centered the message to center of datagrid

### PR DESCRIPTION
**Pull request by GauravMenezes-Professional**
Changes in this PR:

-> Placeholder text is now "No items found!" instead of "We couldn't find any items!"
-> Centered icon in empty container
-> Updated story to reflect the same.

obsdef-10007-datagrid-related-centering

### CODE REVIEW

File: src/datagrid/DataGrid.stories.tsx [hunk 1]
```diff
@@ -367,7 +367,7 @@ storiesOf("DataGrid", module)
     ))
     .add("Empty data grid", () => (
         <div style={{width: "80%"}}>
-            <DataGrid columns={normalColumns} footer={defaultFooter} />
+            <DataGrid columns={normalColumns} footer={defaultFooter} style={{height: "70vh"}} />
```
Comment (maheshkulkarni4) [inline src/datagrid/DataGrid.stories.tsx:370]: Why is this change needed ? Datagrid is used in many places, check the impact.

Comment (GauravMenezes-Professional) [reply src/datagrid/DataGrid.stories.tsx:370]: This is just for the empty DataGrid story to emphasize the center. This will not affect DataGrid in any manner.

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM

**Comment by GauravMenezes-Professional** [comment]
I do not have write access, could anyone with write access please merge if changes are approved?

**Review by psivakumaran — APPROVED** [review]

File: src/datagrid/DataGrid.tsx [hunk 2]
```diff
@@ -1092,7 +1094,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     // Function to create placeholder for empty datagrid
     private buildEmptyPlaceholder(): React.ReactElement {
         const {itemText} = this.state;
-        const placeholderText = "We couldn't find any " + itemText + " !";
+        const placeholderText = "No " + itemText + " found!";
```
Comment (akate1) [inline src/datagrid/DataGrid.tsx:1097]: is this change requested by someone ? We developed this message as per VMware clarity

**Review by akate1 — COMMENTED** [review]
@GauravMenezes-Professional Please update package version in Package.json file
